src/lib/support/BUILD.gn

```diff
@@ -167,6 +167,10 @@ source_set("testing") {
   ]
 }
 
+source_set("linked-list") {
+  sources = [ "LinkedList.h" ]
```

Comment:
Can't it be part of `support`?

Reply:
I thought support pulls in a lot of things. For this particular case, sure -app is very high level.

Is there a reason you would prefer support instead of a separate source set? I had a hard time deciding.

Reply:
I just wonder if having such a fine granularity (source set per file or a few files) won't needlessly complicate our build scripts. I see your point that support currently has too many deps, but on the other hand, do we expect to build any executable without pulling support in some way? Ideally, we should probably gravitate towards leaving only dep-free stuff in support and extracting others into other directories.

Reply:
Updated to pull it into support.
Eventually we may want to split out `containers` or something (I see we have iterators, linkedlist ... maybe there will be others).